src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/ResultsViewExpansion.cs

```diff
@@ -176,5 +232,56 @@ private EvalResultDataItem CreateResultsViewRow(DkmInspectionContext inspectionC
                 editableValue: null,
                 inspectionContext: inspectionContext);
         }
+
+        private EvalResultDataItem CreateResultsViewRow(DkmInspectionContext inspectionContext, string name, Type declaredType, DkmClrValue value, Formatter formatter)
```

Comment:
That makes sense. I'll investigate that change separately from the rest of this change though.